src/Canvanoid.js

```diff
@@ -17,39 +17,93 @@ export default class Canvanoid {
 					  delta: null }
 
 	    this.messagePanel = null;
-		this.score = null;
+		this.instructionsPanel = null;
+		this.scorePanel = null;
 
 		this.state = null; 
 		this.board = null;
 		this.balls = null;
 		this.vaus = null; 
+		this.pause = null;
 	}
 	
 	start() {	
+		// Game Elements
 		this.state = new State();
-		
 		this.board = new Board();
 		this.board.setStage(this.state.stage);
 
-		this.balls = [ new Ball(this.board.x + this.board.w/2, this.board.y + this.board.h/2 + 100) ];
-		this.vaus = new Vaus(this.board.x + this.board.w/2 - 50, 
-              				 this.board.y + this.board.h - 50);
-		this.vaus.start();
+		this.balls = [ new Ball(this.board.position.x + this.board.width/2, this.board.position.y + this.board.height/2 + 100) ];
+		this.vaus = new Paddle(this.board.position.x + this.board.width/2 - 50, 
+              				   this.board.position.y + this.board.height - 50);
```

Comment:
@brauliodiez Fixed this adding two functions: one for putting the game objects in their initial positions, and another for the panels. I chose this way because the game objects (balls and vaus) are always reset together, so I figured I could just have a function for both of them. Same for the panels.